notebooks/acs-dev-v1-cleanup.ipynb

```diff
@@ -54,7 +54,68 @@
     "\n",
     "# would probably be good to test installing package and running this\n",
     "# jupyter notebook directly from command line in a fresh clone, no\n",
-    "# pycharm at all\n"
+    "# pycharm at all"
```

Comment:
I am not sure if I fully understand the purpose of this notebook. 

It sort of feels like it might belong in a getting start for developers guide is that how you intend for it to be used? And since pycharm requires a license should the instructions be set up for any general python console? 

After the line of code 
```
x = stitches.fx_pangeo.fetch_pangeo_table()
```
 Which shows that the stitches functions are working I unsure are to what the purpose of the rest of the code is, could you explain what you are doing/why you are doing it? Will that be important for all users to do as well? Also, the out_dir is hard coded to your machine, which causes an error, should this be a relative path so that it works on other machines as well?

Reply:
Also is the .DS_Store file supposed to be committed? Should we add .DS_Store to the git ignore file?

Reply:
Ah good points. The notebook is more just what I'm using as I'm checking things and testing things and I didn't want to re-make it every time I branched from `dev`. I was envisioning just a final commit deleting it before merging dev into main, but this is cleaner. 

Re adding .DS_Store file to the git ignore, Yes I think that would be super good and I think me adding it to the gitignore on this branch and pushing the commit but I am super terrible at correctly adding things to git ignore, do you have a sample for how to correctly do it  for this specifically that I can copy? I can figure it out through trial and error if not